package.json

```diff
@@ -55,6 +52,13 @@
     "react-native": "0.72.5",
     "@types/react-native": "0.72.3",
     "react-i18next": "11.18.6",
+    "@aries-framework/anoncreds": "0.4.0",
+    "@aries-framework/anoncreds-rs": "0.4.0",
+    "@aries-framework/askar": "0.4.0",
+    "@aries-framework/core": "0.4.0",
+    "@aries-framework/indy-sdk-to-askar-migration": "0.4.0",
+    "@aries-framework/indy-vdr": "0.4.0",
+    "@aries-framework/react-native": "0.4.0",
     "@aries-framework/core@0.3.3": "patch:@aries-framework/core@npm:0.3.3#./.yarn/patches/@aries-framework-core-npm-0.3.3-dd6486de3d.patch",
     "@aries-framework/react-hooks@^0.4.2": "patch:@aries-framework/react-hooks@npm:0.4.2#./.yarn/patches/@aries-framework-react-hooks-npm-0.4.2-84b7eb8764.patch",
     "@aries-framework/react-native@0.4.0": "patch:@aries-framework/react-native@npm:0.3.3#./.yarn/patches/@aries-framework-react-native-npm-0.3.3-bb75ece22d.patch",
```

Comment:
Is there no issue with having two resolutions for different versions of the same dep? ie. core 0.3.3 vs core 0.4.0, react-native 0.3.3 vs 0.4.0

Reply:
Agreed. line 58 (`"@aries-framework/core": "0.4.0"`) and line 62 (`"@aries-framework/core@0.3.3": "patch:@..."`) are conflicting with each other. That means the patch won't apply to core 0.4.0